charts/seed-monitoring/charts/grafana/dashboards/owners/node-pool-dashboard.json

```diff
@@ -185,15 +221,218 @@
       "bars": false,
       "dashLength": 10,
       "dashes": false,
+      "datasource": null,
+      "description": "Shows the average CPU utilization of all nodes in the selected worker group(s).",
+      "fieldConfig": {
+        "defaults": {},
+        "overrides": []
+      },
+      "fill": 0,
+      "fillGradient": 0,
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 0,
+        "y": 7
+      },
+      "hiddenSeries": false,
+      "id": 18,
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "nullPointMode": "null",
+      "options": {
+        "alertThreshold": true
+      },
+      "percentage": false,
+      "pluginVersion": "7.5.16",
+      "pointradius": 2,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "avg by (label_worker_gardener_cloud_pool) (\nsum by (node) (rate(node_cpu_seconds_total{mode!~\"idle|iowait|steal\"}[5m])) / sum by (node) (kube_node_status_capacity{resource=\"cpu\", unit=\"core\"}) * 100\n+ on (node) group_right ()\n  sum by (label_worker_gardener_cloud_pool, node) (\n    0 * kube_node_labels{label_worker_gardener_cloud_pool=~\"$worker_group\"}\n  )\n)",
```

Comment:
- For the rate interval, please use `$__rate_interval` instead of `5m`.

  See https://grafana.com/blog/2020/09/28/new-in-grafana-7.2-__rate_interval-for-prometheus-rate-queries-that-just-work/ for more details.

- I double checked the expression that we talked about and I suggested as well: `mode!~"idle|iowait|steal"`. I think it needs to be improved. There are 8 CPU mode categories in the node_cpu_seconds_total metric:

```
idle
iowait
irq
nice
softirq
steal
system
user
```

  iowait is a subset of idle, nice is a subset of user. So to get the CPU usage, we need to add up `system+user+irq+softirq` which is the complement of "idle, iowait, nice, steal" (nice was missing above). So instead of `mode!~"idle|iowait|steal"` I think we should rather use `mode=~"system|user|irq|softirq"`. (Some other dashboards are also affected.)

- A nit: the query is broken up into multiple lines but it is not formatted with the new formatting feature of Prometheus.

  optional: copy paste the query into the prometheus UI and format it there. (The Grafana variable `$__rate_interval` has to be removed temporarily.)